test/parallel/test-http-should-keep-alive.js

```diff
@@ -20,11 +20,10 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
+const common = require('../common');
```

Comment:
Just noticed but I don't think this `const common` is actually used? In general, I recommend running `make test -j4` — it will run eslint to make sure there are no small issues like this.